src/main/scala/eventstore/ProjectionsClient.scala

```diff
@@ -22,7 +22,7 @@ import scala.util.Try
  * See : https://github.com/EventStore/EventStore/blob/release-v3.9.0/src/EventStore.ClientAPI/Projections/ProjectionsClient.cs
  */
 protected[this] trait ProjectionsUrls {
-  def createProjectionUrl(name: String, mode: ProjectionMode = Continuous, allowEmit: Boolean = true): String = {
+  protected def createProjectionUrl(name: String, mode: ProjectionMode = Continuous, allowEmit: Boolean = true): String = {
```

Comment:
yeah i thought that it would hide the methods as well but it just hide the trait itself, because the methods are public, when you mix in with the ProjectionsClient then they become public in the ProjectionsClient